src/types.js

```diff
@@ -154,7 +155,7 @@ export type Settings = {
   },
   FIRMWARE_DIRECTORY: string,
   FIRMWARE_REPOSITORY_DIRECTORY: string,
-  LOG_REQUESTS: boolean,
+  LOG_LEVEL: 'debug' | 'error' | 'fatal' | 'info' | 'warn' | 'trace' | string,
```

Comment:
Can you help to solve the problem 

```
  LOG_LEVEL: process.env.LOG_LEVEL || 'info',
```
Fails with : process.env.LOG_LEVEL is not compatible, which is correct !